data/csp/azure/sle12/packages.yaml

```diff
@@ -1,24 +1,20 @@
 packages:
+  _namespace_default_kernel: Null
   _namespace_azure_base:
     package:
       - cloud-init
       - cloud-init-config-suse
       - hyper-v
-      - _attributes:
-          name: kernel-default
-          arch: aarch64
       - python-azure-agent
   _namespace_azure_cryptsetup:
     package:
       - cryptsetup
-  _namespace_azure_kernel:
-    package:
-      - _attributes:
-         name: kernel-default
-         arch: x86_64
   _namespace_azure_cloud_netconfig:
     package:
       - cloud-netconfig-azure
+  _namespace_azure_kernel:
+    package:
+      - kernel-default
```

Comment:
One would think that `_namespace_azure_kernel` should have `kernel-azure` but this was already weird previously.

Reply:
Well it's not the namespace for `kernel-azure` but the namespace for whatever kernel is used in Azure, which is `kernel-default` by default and redefined in settings modules that use `kernel-azure`. But I can see how it may be confusing. ;)